airflow/api/common/experimental/get_dag_runs.py

```diff
@@ -25,12 +25,13 @@
 from airflow.models import DagRun
 
 
-def get_dag_runs(dag_id, state=None, run_url_route='Airflow.graph'):
+def get_dag_runs(dag_id, dagrun_id, state=None, run_url_route='Airflow.graph'):
```

Comment:
i think it would be helpful to have this as a kwarg instead. if it's none then we can get all the dagruns instead of being only able to find a specific dagrun. this way the existing dag_runs endpoint will still work and return all the dag runs for a given DAG id

Reply:
Hey, so the existing endpoint does return all the dag_runs if dagrun_id is none:

https://github.com/Affirm/airflow/blob/371a5d583a460d6676ca105460c956b4c19bcaa5/airflow/api/common/experimental/get_dag_runs.py#L43

https://github.com/Affirm/airflow/blob/2379061040861d9a90225e815f6cf44067ebdfee/airflow/models/dagrun.py#L144

Reply:
+1 to kwarg here!